### PR TITLE
Install the Service Points feature

### PR DESCRIPTION
**Pull request by romanhavrylko**
Epic: https://spryker.atlassian.net/browse/CC-26569
Ticket: https://spryker.atlassian.net/browse/CC-28588

### CODE REVIEW

File: _includes/pbc/all/install-features/202304.0/install-the-service-points-feature.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Make sure that the following modules have been installed:
 |------------------------------------|--------------------------------------------------------|
 | ProductOfferServicePoint           | vendor/spryker/product-offer-service-point             |
 | ProductOfferServicePointDataImport | vendor/spryker/product-offer-service-point-data-import |
+| ProductOfferServicePointStorage    | vendor/spryker/product-offer-service-point-storage     |
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/install-features/202304.0/install-the-service-points-feature.md:31]: this module is mapped to another feature
see https://release.spryker.com/features/view/145

**Review by DmitryLymarenko — DISMISSED** [review]

**Review by philinecao — APPROVED** [review]

**Review by VadymSachenko — CHANGES_REQUESTED** [review]

File: _includes/pbc/all/install-features/202304.0/install-the-product-offer-service-points-feature.md [hunk 2]
```diff
@@ -0,0 +1,582 @@
...
+
+Follow the steps below to install the Service Points feature core.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME           | VERSION          | INTEGRATION GUIDE                                                                                                                                                                        |
+|----------------|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Product Offer  | {{site.version}} | [Product Offer feature integration](/docs/pbc/all/offer-management/{{site.version}}/marketplace/install-and-upgrade/install-features/install-the-marketplace-product-offer-feature.html) |
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-product-offer-service-points-feature.md:15]:
```suggestion
| Product Offer  | {{site.version}} | [Product Offer feature integration](/docs/pbc/all/offer-management/{{page.version}}/marketplace/install-and-upgrade/install-features/install-the-marketplace-product-offer-feature.html) |
```

File: _includes/pbc/all/install-features/202304.0/marketplace/install-the-marketplace-product-offer-service-points-feature.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+Follow the steps below to install the Marketplace Product Offer + Product Offer Service Points feature core.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME                         | VERSION          | INTEGRATION GUIDE                                                                                                                                                                               |
+|------------------------------|------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Marketplace Product Offer    | {{page.version}} | [Marketplace Product Offer feature integration](spryker-docs/docs/pbc/all/offer-management/{{page.version}}/marketplace/install-and-upgrade/marketplace-product-offer-feature-integration.html) |
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/marketplace/install-the-marketplace-product-offer-service-points-feature.md:15]:
```suggestion
| Marketplace Product Offer    | {{page.version}} | [Marketplace Product Offer feature integration](/docs/pbc/all/offer-management/{{page.version}}/marketplace/install-and-upgrade/marketplace-product-offer-feature-integration.html) |
```

File: _includes/pbc/all/install-features/202304.0/marketplace/install-the-marketplace-product-offer-service-points-feature.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+| Product Offer Service Points | {{page.version}} | [Product Offer Service Points feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/install-the-product-offer-service-points-feature.html)                            |
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/marketplace/install-the-marketplace-product-offer-service-points-feature.md:16]:
```suggestion
| Product Offer + Service Points | {{page.version}} | [Install Product Offer + Service Points feature](/docs/scos/dev/feature-integration-guides/{{page.version}}/install-the-product-offer-service-points-feature.html)                            |
```

File: _includes/pbc/all/install-features/202304.0/install-the-product-offer-service-points-feature.md [hunk 5]
```diff
@@ -0,0 +1,569 @@
...
+     */
+    protected function getDataImporterPlugins(): array
+    {
+        return [
+            new ProductOfferServiceDataImportPlugin(),
+        ];
+    }
+}
+```
+</details>
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-product-offer-service-points-feature.md:180]:
```suggestion
```

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**